cylc/rose/entry_points.py

```diff
@@ -98,6 +95,9 @@ def rose_fileinstall(dir_=None, opts=None, dest_root=None):
         dest_root (string or pathlib.Path)
 
     """
+    dir_ = pathchecker(dir_)
+    dest_root = pathchecker(dest_root)
```

Comment:
Can do `Path(dest_root)` (Path accepts Path objects as arguments).

Reply:
I thought so, but had a few problems getting this working. Fixed now.